### DISCUSSION TITLE
How can I run an app as an administrator

### DISCUSSION THREAD

**Discussion by frisk099** [Whisky-App/Whisky#695] [Q&A]
How can I run an app as an administrator. Becaus when I runned Epic game the error SU-PQR1603 appeared and when i searched what does it mean, they sad you have to run the app as an administrator.
Please help !

**Comment by hak-e** [comment]
Hi
I managed to go a little further, but the launcher still crashes in the end(
-SkipBuildPatchPrereq

![Screenshot 1](https://github.com/Whisky-App/Whisky/assets/115734216/62e5c6d4-32e0-4cc6-a00d-7405e01487d2)
![Screenshot 2](https://github.com/Whisky-App/Whisky/assets/115734216/81271423-3439-40d5-8bcd-7e9ea0d6d79e)
